centreon/tests/performanceTesting/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "test-perf",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "commander": "^13.1.0",
+    "dotenv": "^16.4.7",
+    "js-yaml": "^4.1.0",
+    "mysql2": "^3.12.0",
+    "node-fetch": "^3.3.2"
```

Comment:
If you are running on nodeJS 20, you don't need to install node_fech as it's already included inside NodeJS: https://nodejs.org/dist/latest-v20.x/docs/api/globals.html#fetch